src/overlays/actors/ovl_Bg_Sinkai_Kabe/z_bg_sinkai_kabe.c

```diff
@@ -66,11 +68,11 @@ void BgSinkaiKabe_Init(Actor* thisx, GlobalContext* globalCtx) {
     sCurrentPythonIndex++;
 
     if (!(gSaveContext.save.weekEventReg[13] & 1)) {
-        child = Actor_SpawnAsChild(&globalCtx->actorCtx, &this->dyna.actor, globalCtx, ACTOR_EN_DRAGON, pos.x, pos.y,
-                                   pos.z, 0, this->dyna.actor.world.rot.y, 1, this->dyna.actor.params);
-        this->deepPythonChild = child;
+        this->deepPythonChild =
```

Comment:
I'm not really sure it needs `Child` in the name now.

Reply:
Sure, fair enough. I wrote it like that because eventually, `EnDragon* dragon` will probably become `DeepPython* python`. Which will result in the code looking like `DeepPython* python = (DeepPython*) this->deepPython`, which looks a bit silly. However, that's a problem way in the future, so there's no real reason to optimize for it now. I'll just drop `child` for now.